test/DotNetIsolated60/HttpRequestDataFunction.cs

```diff
@@ -5,16 +5,16 @@
 
 namespace DotNetIsolated60
 {
-    public class Function1
+    public class HttpRequestDataFunction
     {
         private readonly ILogger _logger;
 
-        public Function1(ILoggerFactory loggerFactory)
+        public HttpRequestDataFunction(ILoggerFactory loggerFactory)
         {
-            _logger = loggerFactory.CreateLogger<Function1>();
+            _logger = loggerFactory.CreateLogger<HttpRequestDataFunction>();
```

Comment:
Since we're updating, let's not use the factory pattern and switch to 

```csharp
private readonly ILogger<CloudEventSamples> _logger;

        public CloudEventSamples(ILogger<CloudEventSamples> logger)
        {
            _logger = logger;
        }
```